aten/src/ATen/native/RNN.cpp

```diff
@@ -340,8 +344,9 @@ struct QuantizedCellParamsDynamic : public CellParamsBase {
     std::vector<c10::intrusive_ptr<LinearPackedParamsBase>>
         packed_params_to_serialize{packed_w_ih, packed_w_hh};
 
+    string op_name = reduce_range_ ? "quantized_dynamic_bc" : "quantized_dynamic";
```

Comment:
Why call this quantized_dynamic_bc? This is the newer version that has reduce_range=True.